velox/expression/PeeledEncoding.h

```diff
@@ -158,19 +158,38 @@ class PeeledEncoding {
   /// Takes a set of outer rows (rows defined over the peel) and a functor that
   /// is executed for each outer row which does not map to a null row.
   /// The outer and its corresponding inner row is passed to the functor.
-  void applyToNonNullInnerRows(
-      const SelectivityVector& outerRows,
-      std::function<void(vector_size_t, vector_size_t)> func) const {
-    auto indices = wrap_ ? wrap_->as<vector_size_t>() : nullptr;
-    auto wrapNulls = wrapNulls_ ? wrapNulls_->as<uint64_t>() : nullptr;
-    outerRows.applyToSelected([&](auto outerRow) {
-      // A known null in the outer row masks an error.
-      if (wrapNulls && bits::isBitNull(wrapNulls, outerRow)) {
-        return;
+  template <typename F>
+  void applyToNonNullInnerRows(const SelectivityVector& outerRows, F&& func)
```

Comment:
@Yuhta Can you please update the documentation in this header file with the new possible outputs and those that are now outdated?

Reply:
They still all hold I think, just when the dictionary base is too large we will do flattening.  I can update the doc to add this.

Reply:
https://github.com/facebookincubator/velox/pull/10701